packages/ketcher-core/src/application/editor/actions/rgroupAttachmentPoint.ts

```diff
@@ -64,8 +64,8 @@ function fromRGroupAttachmentPointAddition(attpnt, atomId: number) {
 function fromRGroupAttachmentPointsDeletion(struct: Struct, atomId: number) {
   const action = new Action();
   const attachmentPointsToDelete =
-    RGroupAttachmentPoint.getRGroupAttachmentPointsByAtomId(atomId, struct);
-  attachmentPointsToDelete.forEach((_attachmentPoint, id) => {
```

Comment:
Are you sure that first argument must be removed?  Than, maybe, rename "id" to "element" or "item", to be explicit that item (not index) is used

Reply:
Yes, I'm sure the first argument `_attachmentPoint` can be removed. The previous `getRGroupAttachmentPointsByAtomId` returns a `Pool`, but the current one returns an array which contains `id`s of RGroupAttachmentPoints.
I think I can rename `id` to `rgroupAttachmentPointId`.

Reply:
Okay, cool! Thanks for clarifying=)